app/src/androidTest/java/com/duckduckgo/app/statistics/VariantManagerTest.kt

```diff
@@ -32,14 +32,14 @@ class VariantManagerTest {
     @Test
     fun serpControlVariantIsInactiveAndHasNoFeatures() {
```

Comment:
This test name no longer matches beahvior. Let's save the next dev some pain and change it to something generic so it doesn't need to be changed by them too. Maybe `serpControlVariantHasExpectedWeightAndNoFeatures`?